src/components/PostComment.jsx

```diff
@@ -0,0 +1,80 @@
+import { useState } from "react";
+import { fetchCommentsByArticleId, postCommentByArticleId } from "../utils";
+import BeatLoader from "react-spinners/BeatLoader";
+
+export default function PostComment({ article_id, setComments }) {
+  const [newComment, setNewComment] = useState("");
+  const [isLoading, setIsLoading] = useState(false);
+  let [color, setColor] = useState("#ffffff");
```

Comment:
Any reason this is let rather than const?

Reply:
not really. 😅 sorry. This is from when I copied the code from the docs.